templates/User/edit.html.twig

```diff
@@ -66,11 +66,17 @@
                                 {{ form_widget(edit_form._token) }}
                             </form>
                             <p>
-                                {{ form_start(delete_form) }}
-                                    <button {% if app.user.id == user.id %}disabled="disabled"{% endif %} onclick="return confirm('{{ 'user.form.delete_confirm'|trans|escape('js') }}')" type="submit" class="btn waves-effect waves-light red">{{ 'user.form.delete'|trans }}</button>
-                                {{ form_end(delete_form) }}
+                                {% if is_granted('DELETE', user) %}
+                                    {{ form_start(delete_form) }}
+                                        <button onclick="return confirm('{{ 'user.form.delete_confirm'|trans|escape('js') }}')" type="submit" class="btn waves-effect waves-light red">{{ 'user.form.delete'|trans }}</button>
+                                    {{ form_end(delete_form) }}
+                                {% else %}
+                                    <button disabled="disabled" type="button" class="btn waves-effect waves-light red">{{ 'user.form.delete'|trans }}</button>
+                                {% endif %}
                             </p>
-                            <p><a class="waves-effect waves-light btn blue-grey" href="{{ path('user_index') }}">{{ 'user.form.back_to_list'|trans }}</a></p>
+                            {% if is_granted('LIST_USERS') %}
+                                <p><a class="waves-effect waves-light btn blue-grey" href="{{ path('user_index') }}">{{ 'user.form.back_to_list'|trans }}</a></p>
+                            {% endif %}
```

Comment:
@j0k3r @nicosomb @Kdecherf need your opinion on this: should I strictly wrap every link everywhere like now? or should we assume that if you got access to the edit template it's likely that you have access to the list already, and so no need to check?
The 2 approach are valid IMO, at work we use the later for less `is_granted` in case some are complex to compute, but I feel the former is "more secure" in case of code copy/paste or future usages that will make the assumption wrong

Reply:
I think we can assume that if the user can edit a user, it has access to the list as well.

Reply:
@nicosomb @Kdecherf any other opinion? meanwhile I implemented @j0k3r what wants :)

Reply:
Seems reasonable

Reply:
I just removed the is_granted for LIST_USERS on new and edit pages